dist/get.kobman.io

```diff
@@ -20,14 +21,11 @@ kobman_src_folder="${KOBMAN_DIR}/src"
 kobman_tmp_folder="${KOBMAN_DIR}/tmp"
 kobman_stage_folder="${kobman_tmp_folder}/stage"
 kobman_zip_file="${kobman_tmp_folder}/kobman-${KOBMAN_VERSION}.zip"
-kobman_zip_tests="${kobman_tmp_folder}/kobman-test.zip"
-kobman_tests_folder="${KOBMAN_DIR}/tests"
 kobman_env_folder="${KOBMAN_DIR}/envs"
 kobman_stage_folder="${kobman_tmp_folder}/stage"
```

Comment:
where is stage, archives folder used?

Reply:
Once these stage, archives are commented the kobman-latest.zip download is not happening